python/python3_12/PythonParser.g4

```diff
@@ -427,8 +432,7 @@ maybe_star_pattern
     | pattern;
 
 star_pattern
-    : '*' pattern_capture_target
-    | '*' wildcard_pattern;
+    : '*' NAME;
```

Comment:
There's not a single test for this grammar change.
```
10/21-09:31:22 ~/issues/g4-4280/python/python3_12/Generated-CSharp-0
$ ./bin/Debug/net8.0/Test.exe ../examples/*.py -tree 2>&1 | grep star_pattern
10/21-09:32:57 ~/issues/g4-4280/python/python3_12/Generated-CSharp-0
$
```

Reply:
I added test, thanks.